CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+
+* Problems found during a new developer environment setup.
+* Gaps in our Quickstart Guide or documentation.
+* Bugs in our automation scripts.
+
+If anything doesn't make sense, or doesn't work when you run it, please open a
+[issue](https://github.com/rpm-software-management/dnf5/issues/new) and let us know!
+If you file a bug on Fedora you could open a bug on
+[Bugzilla](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=dnf5).
+
+## Ways to Contribute
+
+We welcome many different types of contributions including:
+
+* [New features](https://github.com/rpm-software-management/dnf5/labels/RFE)
+* [Bug fixes](https://bugzilla.redhat.com/buglist.cgi?bug_status=__open__&product=Fedora&component=dnf5)
+* [Documentation](https://dnf5.readthedocs.io)
+* [Issue Triage](https://github.com/rpm-software-management/dnf5/issues)
+* [Copr Builds](https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/), [CI/CD](https://github.com/rpm-software-management/dnf5/actions)
+* [Communications](#ask-for-help) / [GH Discussions](https://github.com/rpm-software-management/dnf5/discussions)
+
+## Find an Issue
+
+We have good first issues for new contributors and help wanted issues suitable
+for any contributor. Issues labeled as
+[good first issue](https://github.com/rpm-software-management/dnf5/labels/good%20first%20issue)
+are issues suitable for someone who isn't a core maintainer and wants to get familiar with the codebase.
+
+Sometimes there won’t be any issues with these labels. That’s ok! There is
+likely still something for you to work on. If you want to contribute but you
+don’t know where to start or can't find a suitable issue, you can ask for help by reaching out on one of our
+[channels](#ask-for-help).
+
+If you see an issue that you'd like to work on, please post a comment saying
+that you want to work on it and assign yourself.
+Something like "I want to work on this" is fine.
+
+## Ask for Help
+
+The best way to reach us with a question when contributing is to ask on:
+
+* The original github issue
+* Our mailing list [sst-cs-software-management@redhat.com](mailto:sst-cs-software-management@redhat.coms)
+* On IRC [#dnf](irc:/chat.libera.org/#dnf)
+
+## Pull Request Lifecycle
+
+Once you have a contribution ready, follow these steps:
+- Submit the PR
+  - Set the [labels](https://github.com/rpm-software-management/dnf5/labels) accordingly.
+  - If you need help while working on it you could set
+    [work in progess](https://github.com/rpm-software-management/dnf5/labels/work%20in%20progress)
+    and wait for the PR to be triaged.
+  - If you are ready for a code review by one of the maintaineres set the label
+    [ready for review](https://github.com/rpm-software-management/dnf5/labels/ready%20for%20review).
+- Triaging happens usually on Mondays or Thursdays
+- Review!
+  - Discuss the pr with one of the maintainers.
+  - Some changes might be requested.
+  - Review again.
+- Merge!
```

Comment:
```suggestion
Once you have a contribution ready, complete the following steps:
- Submit a pull request (PR).
  - Set the necessary [labels](https://github.com/rpm-software-management/dnf5/labels).
  - If you need help while working on the PR, set the
    [_work in progess_](https://github.com/rpm-software-management/dnf5/labels/work%20in%20progress) label
    and wait for the PR to be triaged.
  - If you are ready for a code review by one of the maintainers, set the
    [_ready for review_](https://github.com/rpm-software-management/dnf5/labels/ready%20for%20review) label.
    Triaging usually happens on Mondays or Thursdays.
- Review the PR.
  - Discuss the PR with one of the maintainers. Note that some changes might be requested.
  - Review again.
- Merge the PR.
```

I would also use the numbered list for the main steps. This ensures that folks go step by step and do not perform these steps in a different order (even though it is obvious that those steps should be performed one by one :-D ) But this is just a suggestion :-)

Reply:
I like the suggestion, I'll make the change